CHANGELOG.md

```diff
@@ -39,6 +40,17 @@
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 
+## [v1.13.0]
+
+> Release date: TBD
+
+### Fixes
+
+- Since Konnect doesn't have any concept of workspaces, decK should return
+  a meaningfull error message whenever running against Konnect with a config file
+  having the `_workspace` entry or the `--workspace` flag is used.
```

Comment:
```suggestion
- Improve error message when workspace is used in Konnect mode
```

A changelog is for the end-user. They need to be notified of changes, not of 'why', 'how', 'when' at this level of detail.